_inc/client/components/jumpstart/index.jsx

```diff
@@ -29,55 +30,115 @@ const JumpStart = React.createClass( {
 					{ __( 'Jump Start your Website' ) }
 				</h2>
 				<Card className="jp-jumpstart__cta-container">
-					{ this.props.jumpstarting( this.props ) ? <Spinner /> : null }
 					<Card className="jp-jumpstart__cta">
+						{ this.props.jumpstarting( this.props ) ? <Spinner /> : null }
 						<p className="jp-jumpstart__description">
 							{ __( "Quickly enhance your site by activating Jetpack's recommended features." ) }
 						</p>
 						<Button primary={ true } onClick={ this.props.jumpStartActivate }>
 							{ __( 'Activate Recommended Features' ) }
 						</Button>
 					</Card>
-					<Card className="jp-jumpstart__features">
+					<FoldableCard
+						className="jp-jumpstart__features"
+						clickableHeaderText={ true }
+						subheader="Learn more"
```

Comment:
Needs translation wrapper. Also, when I click this, nothing happens -- should it be opening up the rest of the UI or something? I don't see any of the UI in your screenshots, except for the top button basically.